packages/app/src/components/App.js

```diff
@@ -152,6 +174,19 @@ export class App extends Component {
                 e.ctrlKey === true &&
                 this.props.setCurrentFromUi(this.props.ui)
         )
+
+        const t = this
```

Comment:
Not necessary as you are using an arrow function in the event listener? You should be able to just do this.props.visualization etc inside the arrow function.

Reply:
I thought the same but it didn't work, I tried multiple combinations but this was the only way to make it work....

Reply:
Weird, seems to work here

Reply:
I get t === this // true

Reply:
Weird, also working for me now. Before I got `this` but it was different to the `this` I wanted. Maybe there was a cache issue when I tested ⚡️ anyway, updated!